components/skus/browser/rs/lib/src/sdk/credentials/fetch.rs

```diff
@@ -134,8 +139,9 @@ where
                                 match item_creds.state {
                                     CredentialState::GeneratedCredentials
                                     | CredentialState::SubmittedCredentials => {
-                                        // we have generated, or performed submission, reuse the creds
-                                        // we created for signing.
+                                        // we have generated, or performed submission, reuse the
+                                        // creds we created
+                                        // for signing.
```

Comment:
Looks like the formatter failed here. Comment should still be on two lines.

Reply:
Fixed in https://github.com/brave/brave-core/pull/19398/commits/760fa85eb4d2e800e57315ba36927e5043afcb3a